### ISSUE TITLE
re generates wrong JSON

### ISSUE THREAD

**Issue by pkuzmin** [katef/libfsm#186]
In this case, there is not enough comma.
```
./build/bin/re -pl json "^[abc]$"
{
        "states": [
                {
                        "end": false,
                        "edges": [
                                { "char":  "a", "to": 1},
                                { "char":  "b", "to": 1}
                                { "char":  "c", "to": 1}
                        ]
                },
                {
                        "end": true,
                        "edges": [
                        ]
                }
        ],
        "start": 0
}
```
But here everything is fine.
```
./build/bin/re -pl json "^[ace]$"
{
        "states": [
                {
                        "end": false,
                        "edges": [
                                { "char":  "a", "to": 1},
                                { "char":  "c", "to": 1},
                                { "char":  "e", "to": 1}
                        ]
                },
                {
                        "end": true,
                        "edges": [
                        ]
                }
        ],
        "start": 0
}
```

**Comment by katef** [comment]
Thank you very much!
Addressed in 19f5388a

**Comment by katef** [comment]
Incidentally, the json output there is a quite literal representation of an FSM, with every transition spelled out explicitly.

That might be exactly what you wanted, but depending on what you're looking for, you might also be interested in the `re -pl irjson` output. That comes from a later phase of processing (just before code generation), and contains various optimisations, for example folding together ranges of adjacent transitions.

See #94 for a quick overview of the IR nodes.
